bin/webpack-dev-server.js

```diff
@@ -310,8 +314,15 @@ function processOptions(wpOpt) {
 	if(argv["compress"])
 		options.compress = true;
 
-	if(argv["open"] || argv["open"] === "")
-		options.open = argv["open"];
+	if(argv["open"]) {
```

Comment:
I added it when I switched it from a boolean to a string, but now that it's back to a boolean we don't need that check.

But that did make me think about the same check on the open-page arg, so I changed it around so that I don't need that check anymore by making the open-page argument require a value. I think it's much cleaner now.